Remotion/ObjectBinding/Web/UI/Controls/BocEnumValue.cs

```diff
@@ -322,14 +324,19 @@ protected object GetValue ()
     /// <remarks>
     /// <para>Setting the value via this method does not affect the control's dirty state.</para>
     /// </remarks>
-    protected void SetValue (object value)
+    protected void SetValue (object? value)
     {
       _value = value;
 
       if (Property != null && _value != null)
-        _enumerationValueInfo = Property.GetValueInfoByValue (_value, GetBusinessObject());
+      {
+        var businessObject = Assertion.IsNotNull (GetBusinessObject(), "GetBusinessObject() must not return a null value.");
+        _enumerationValueInfo = Property.GetValueInfoByValue (_value, businessObject);
```

Comment:
the businessObject parameter is nullable now.
```suggestion
        _enumerationValueInfo = Property.GetValueInfoByValue (_value, GetBusinessObject());
```